src/main/java/com/sublinks/sublinksapi/slurfilter/services/SlurFilterService.java

```diff
@@ -36,11 +36,11 @@ public void updateOrCreateLemmySlur(String regex) {
 
   public SlurFilter getLemmySlurFilter() {
 
-    SlurFilter slurFilter = slurFilterRepository.findAll().stream().findFirst().orElse(null);
+    SlurFilter slurFilter = slurFilterRepository.findById(1L).orElse(null);
```

Comment:
Same here

Reply:
As if its getting deleted lemmy support is dropped. This should ensure, it wont make any side effects.

Reply:
I agree, we shouldn't be hard coding db keys.

Reply:
Ok this could harm a lot of a instance if there is an error.